deflate.c

```diff
@@ -181,24 +184,20 @@ static const config configuration_table[10] = {
   } while (0)
 
 /* ========================================================================= */
-int32_t Z_EXPORT PREFIX(deflateInit_)(PREFIX3(stream) *strm, int32_t level, const char *version, int32_t stream_size) {
-    return PREFIX(deflateInit2_)(strm, level, Z_DEFLATED, MAX_WBITS, DEF_MEM_LEVEL, Z_DEFAULT_STRATEGY, version, stream_size);
+/* Hide this function definition when ZLIB_COMPAT */
+#ifdef ZLIB_COMPAT
```

Comment:
This stuff might be better as a macro..
```
#ifdef ZLIB_COMPAT
#  define ZNG_EXPORT 
#else
#  define ZNG_EXPORT Z_EXPORT
#endif
```
Then can just use `ZNG_EXPORT` or whatever you want to call it in the function definition.

Reply:
Yep, much better 👍

Reply:
Fixed.